### PR TITLE
Fix fetch timing

### PR DESCRIPTION
**Pull request by clemmy**

### CODE REVIEW

**Comment by clemmy** [comment]
@jasonf7 updated bruh

File: app/containers/Session.js [hunk 1]
```diff
@@ -37,6 +37,56 @@ class Session extends Component {
...
+              <br/>
+            </CardText>
+          </Card>
+        )
+      case 'Doubles':
+        return (
+          <Card key={game.courtId} style={cardStyle}>
+            <CardTitle title={game.courtName} subtitle={titleText} />
+            <CardText>
+              {game.team1.length ? game.team1[0].name : '?'}
```
Comment (c9dong) [inline app/containers/Session.js]: this can be put into a format function

File: app/containers/Session.js [hunk 2]
```diff
@@ -37,6 +37,56 @@ class Session extends Component {
...
+              <br/>
+              {game.team1.length > 1 ? game.team1[1].name : '?'}
+            </CardText>
+            <CardText style={{color: 'rgba(0, 0, 0, 0.541176)'}}>
```
Comment (c9dong) [inline app/containers/Session.js:81]: put this in a style file

File: app/reducers/games.js [hunk 3]
```diff
@@ -3,7 +3,72 @@ const initialState = [];
 const gameReducer = (state = initialState, action) => {
   switch (action.type) {
     case 'RECEIVE_MATCH_DATA':
-      return action.games;
+      return action.games.map(g => {
+        if (g.team1.length + g.team2.length === 3) {
```
Comment (c9dong) [inline app/reducers/games.js]: why is the length hardcoded?

File: app/containers/Session.js (hunk 2 above)
Comment (clemmy) [reply app/containers/Session.js:81]: this is copied and pasted from your code bitch

File: app/reducers/games.js (hunk 3 above)
Comment (clemmy) [reply app/reducers/games.js]: what do you mean?

**Review by jasonf7 — APPROVED** [review]

**Pull request merged**